examples/date/moq_communication.md

```diff
@@ -0,0 +1,98 @@
+# MoQ Communication Flow in the date example
+
+This document describes the communication flow between the
+client and server in this example.
+
+## Communication Overview
+
+The communication between the MoQ client and server follows these main steps:
+
+1. **Connection Setup**: The client initiates a connection to the server
+2. **Session Establishment**: Client and server exchange setup messages
+3. **Track Announcement**: Server announces available tracks
+4. **Subscription**: Client subscribes to a specific track
+5. **Data Transfer**: Server sends time updates to the client
+
+## Detailed Message Flow
+
+### 1. Connection Setup
+The client connects to the server using either QUIC or WebTransport as the underlying transport protocol.
+
+### 2. Session Establishment
+- Client sends a `client_setup` message with supported protocol versions and parameters
+- Server responds with a `server_setup` message selecting the protocol version
+
+### 3. Track Announcement
+- Server announces the "clock" namespace
+- Client responds with an `announce_error` message (this appears to be a protocol quirk in this implementation)
+
+### 4. Subscription
+- Client sends a `subscribe` message for the "second" track in the "clock" namespace
+- This is seen in the log as 7365636f6e64, a hex encoding of "second"
```

Comment:
I think the log might have been broken by introducing Qlog. I'll open an issue to look into this.

Reply:
https://github.com/mengelbart/moqtransport/issues/196